.github/workflows/deploy.yaml

```diff
@@ -4,7 +4,7 @@ on:
   # Runs on pushes targeting the `main` branch. Change this to `master` if you're
   # using the `master` branch as the default branch.
   push:
-    branches: [main]
+    branches: [onmax/new-web-client]
```

Comment:
I guess this shouldn't be part of the PR

Reply:
nop, this is just temporarly. GitHub Pages does not allow multiple branches